tox.ini

```diff
@@ -27,16 +27,16 @@ deps =
     pytest-cov
     pytest-html
 commands = pytest \
+	     --asyncio-mode=auto \
           --cov=pytest_operator \
-	      --cov-report=term-missing \
-	      --cov-report=annotate:{toxinidir}/report/unit/coverage-annotated \
-	      --cov-report=html:{toxinidir}/report/unit/coverage-html \
-	      --cov-report=xml:{toxinidir}/report/unit/coverage-xml \
+	     --cov-report=term-missing \
+	     --cov-report=annotate:{toxinidir}/report/unit/coverage-annotated \
+	     --cov-report=html:{toxinidir}/report/unit/coverage-html \
+	     --cov-report=xml:{toxinidir}/report/unit/coverage-xml \
           --cov-config={toxinidir}/tox.ini \
-	      --html={toxinidir}/report/unit/tests/index.html \
-	      --junitxml={toxinidir}/report/unit/junit.xml\
```

Comment:
notice the lack of a `space` following `junit.xml` messed up the new line continuation formatting in tox 4.x

it would have produced a command:

```
--junitxml={toxinidir}/report/unit/junit.xml--asyncio-mode=auto
```

rather than 
```
--junitxml={toxinidir}/report/unit/junit.xml --asyncio-mode=auto
```